tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Supplier Section Update Validation.feature

```diff
@@ -0,0 +1,70 @@
+        | New Line1 | Lower Flat | Rocks Close | Larger Village | Massive Village | Another Town | N Yorks | YO11 1AP | United Kingdom |
+    And the user wants to update the SupplierContact section for the contact
+        | FirstName | LastName | EmailAddress | TelephoneNumber |
+        | NULL      | NULL     | NULL         | NULL            |
+    When the user makes a request to update the supplier with order ID C000014-01
+        | SupplierId | SupplierName     |
+        | Sup3       | Updated Supplier |
+    Then a response with status code 400 is returned
+    And the response contains the following errors
+        | id                      | field           |
+        | FirstNameRequired       | FirstName       |
+        | LastNameRequired        | LastName        |
+        | EmailAddressRequired    | EmailAddress    |
+        | TelephoneNumberRequired | TelephoneNumber |
+
+@4621
+Scenario: 2. Updating a supplier section, and exceeding the maxLength fields, produces the relavent error message
+    Given the user wants to update the SupplierAddress section for the address
+        | Line1     | Line2      | Line3       | Line4          | Line5           | Town         | County  | Postcode | Country        |
+        | New Line1 | Lower Flat | Rocks Close | Larger Village | Massive Village | Another Town | N Yorks | YO11 1AP | United Kingdom |
+    And the user wants to update the SupplierContact section for the contact
+        | FirstName                | LastName                 | EmailAddress                  | TelephoneNumber         |
+        | #A string of length 101# | #A string of length 101# | #A string of length 252#@.com | #A string of length 36# |
+    When the user makes a request to update the supplier with order ID C000014-01
+        | SupplierId | SupplierName     |
+        | Sup3       | Updated Supplier |
+    Then a response with status code 400 is returned
+    And the response contains the following errors
+        | id                     | field           |
+        | FirstNameTooLong       | FirstName       |
+        | LastNameTooLong        | LastName        |
+        | EmailAddressTooLong    | EmailAddress    |
+        | TelephoneNumberTooLong | TelephoneNumber |
+
+@4621
+Scenario: 3. Updating a supplier section, and not providing a correct email address format, produces the relavent error message
+    Given the user wants to update the SupplierAddress section for the address
+        | Line1     | Line2      | Line3       | Line4          | Line5           | Town         | County  | Postcode | Country        |
+        | New Line1 | Lower Flat | Rocks Close | Larger Village | Massive Village | Another Town | N Yorks | YO11 1AP | United Kingdom |
+    And the user wants to update the SupplierContact section for the contact
+        | FirstName | LastName | EmailAddress | TelephoneNumber |
+        | Greg      | Smith    | INVALID      | 23456234521     |

```

Comment:
we could totally make this have various invalid email formats, if you feel so inclined

Reply:
Will push in the next validation checkin